phira/locales/en-US/profile.ftl

```diff
@@ -1,19 +1,19 @@
 
 logout = Logout
-logged-out = Logged out
+logged-out = Logged out.
 
-delete = Delete account
-delete-failed = Failed to delete account
-delete-req-sent = Account deletion request has been submitted
+delete = Delete Account
+delete-failed = Failed to delete account.
+delete-req-sent = Account deletion request has been submitted.
 
-load-user-failed = Failed to fetch user info
-edit-avatar-success = Avatar updated
-edit-avatar-failed = Failed to update avatar
+load-user-failed = Failed to fetch user info.
+edit-avatar-success = Avatar updated.
+edit-avatar-failed = Failed to update avatar.
 
-uploading-avatar = Uploading avatar
+uploading-avatar = Uploading avatar...
 
-load-record-failed = Failed to load records
+load-record-failed = Failed to load scores.
```

Comment:
The captured "Record" actually includes chart info, scores, user info and more. I think "Record" is better than "Score".